christfellowshipchurch/web-app-v2
# Easter 2024

### About
This PR deploys our 2024 Easter page.

### Test Instructions
1. Ensure that all sections look like our Figma mockup.
2. Test that **ALL** major pages are working as aspected and nothing broke. Test Mobile and Desktop view on pages like `events`, `discover`, and the `home` page.

### Screenshots
<img width="1440" alt="Screenshot 2024-02-19 at 9 29 58 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/dc725353-671c-427b-9f5b-d02fb03a68c6">


### Closes Tickets
[CFDP-2915](https://christfellowshipchurch.atlassian.net/browse/CFDP-2915)
[CFDP-2916](https://christfellowshipchurch.atlassian.net/browse/CFDP-2916)
[CFDP-2917](https://christfellowshipchurch.atlassian.net/browse/CFDP-2917)
[CFDP-2918](https://christfellowshipchurch.atlassian.net/browse/CFDP-2918)
[CFDP-2919](https://christfellowshipchurch.atlassian.net/browse/CFDP-2919)


[CFDP-2915]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2915?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CFDP-2916]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2916?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CFDP-2917]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2917?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CFDP-2918]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2918?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CFDP-2919]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2919?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
